docs/source/guide/distributed-preprocessing.rst

```diff
@@ -291,11 +291,11 @@ Step.1 Graph Partitioning
 This step reads the chunked graph data and calculates which partition each node
 should belong to. The results are saved in a set of *partition assignment files*.
 For example, to randomly partition MAG240M-LSC to two parts, run the
-``partition_algo/random.py`` script in the ``tools`` folder:
+``partition_algo/random_partition.py`` script in the ``tools`` folder:
 
 .. code-block:: bash
 
-    python /my/repo/dgl/tools/partition_algo/random.py
+    python /my/repo/dgl/tools/partition_algo/random_partition.py
         --metadata /mydata/MAG240M-LSC_chunked/metadata.json
```

Comment:
We should have a relative consistent argument naming for partition algorithm and dispatch algorithm. The dispatch algorithm uses `--in-dir` and `--out-dir`, so should the partition algorithm here. Please change it in this PR. https://github.com/dmlc/dgl/blob/2e8ae9f980bd0b3db9963626fa8d1cb64f46e4d6/tools/dispatch_data.py#L63-L71

Reply:
Done